pkgs/tools/filesystems/s3backer/default.nix

```diff
@@ -16,6 +16,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook pkg-config ];
   buildInputs = [ fuse curl expat ];
 
+  postPatch = lib.optionalString stdenv.cc.isClang ''
+    substituteInPlace configure.ac --replace \
+      'AC_CHECK_DECLS(fdatasync)' ""
+  '';
+
```

Comment:
`AC_CHECK_DECLS` doesn't work with clang.

Reply:
Can you add a comment with that info?

Reply:
Added.